scripts/build.sh

```diff
@@ -5,7 +5,8 @@ echo @@@@ Prepare to build SKNanaAnalyzer in $SKNANO_BUILDDIR
 rm -rf $SKNANO_BUILDDIR $SKNANO_LIB 
 mkdir $SKNANO_BUILDDIR && cd $SKNANO_BUILDDIR
 
-echo @@@@ camke -DCMAKE_INSTALL_PREFIX=$SKNANO_HOME $SKNANO_HOME
+export CORRECTION_CMAKE_PREFIX=`correction config --cmake`
+echo @@@@ camke -DCMAKE_INSTALL_PREFIX=$SKNANO_HOME $CORRECTION_CMAKE_PREFIX $SKNANO_HOME
```

Comment:
minor typo in `camke` , 
`echo @@@@ cmake -DCMAKE_INSTALL_PREFIX=$SKNANO_HOME $CORRECTION_CMAKE_PREFIX $SKNANO_HOME`